criu/include/cr_options.h

```diff
@@ -134,6 +134,7 @@ struct cr_options {
 	int link_remap_ok;
 	int log_file_per_pid;
 	int pre_dump_mode;
+	int restore_on_demand;
```

Comment:
`bool restore_on_demand;`

Reply:
Using a type of `bool` is more appropriate.

Reply:
You are right, thanks for pointing it out.